specification/dimensions/region.md

```diff
@@ -0,0 +1,30 @@
+# Region
+
+A Region is an isolated geographic area where a resource is provisioned in and/or a service is provided from. Region is commonly used for scenarios like analyzing cost and prices based on where resources are deployed.
+
+The Region column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values.
```

Comment:
Should we say something about the format of the values to ensure they're consistent? 
```suggestion
The Region column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values. Region values MUST be consistent within the provider and MUST be the same values used to indicate the region when provisioning or purchasing the resource or service.
```